runtime/vm/CRIUHelpers.cpp

```diff
@@ -1187,6 +1190,64 @@ toggleSuspendOnJavaThreads(J9VMThread *currentThread, U_8 suspendResumeFlag)
 	}
 }
 
+/**
+ * Get the package/class.method array specified via a system property
+ * -Dopenj9.internal.criu.notcheckpointsafe=package1/class1.method1,package2/class2.method2
+ *
+ * Caller must free the returning structure.
+ *
+ * @param[in] vmThread vmThread token
+ * @param[in/out] classMethodNumber the number of class/method specified
+ */
+static char**
+getNotCheckpointSafeClassMethodArray(J9VMThread *vmThread, UDATA *classMethodNumber)
+{
+	J9JavaVM *vm = vmThread->javaVM;
+	PORT_ACCESS_FROM_JAVAVM(vm);
+	J9VMInitArgs *vmargs = vm->vmArgsArray;
+	char **notcheckpointsafeArray = NULL;
+	/* the rightmost option wins */
+	IDATA argIndex = findArgInVMArgs(PORTLIB, vmargs, STARTSWITH_MATCH, OPENJ9_NOT_CHECKPOINT_SAFE_EQUALS, NULL, FALSE);
```

Comment:
could we just use `vm->internalVMFunctions->getSystemProperty(vm, property, &systemProperty)` to get the sys prop?

Reply:
Good point, use `getSystemProperty()` instead.